src/aura/HH_ContactCard/HH_ContactCard.cmp

```diff
@@ -18,7 +18,7 @@
                                    xlinkHref="/resource/SLDS/1_0_3/assets/icons/standard-sprite/svg/symbols.svg#contact" />
                         </div>
                         <div class="slds-media__body">
-                            <a href="#">{!v.contact.FirstName}&nbsp;{!v.contact.LastName}</a>
+                            <a href="{! '/' + v.contact.Id}" target="_blank">{!v.contact.FirstName}&nbsp;{!v.contact.LastName}</a>
```

Comment:
This works correctly in LEX and Classic in Spring '16.  It was failing in Summer '16 (I think under both LEX and Classic), but testing again this morning it now works in Summer '16.  I'm guessing they refreshed GS0 last night!!